script.js

```diff
@@ -1,5 +1,5 @@
 // Wrap every letter in a span
-console.log('im working now');
+//Your animation is cool but I feel like having it go faster is better to drawing people in
```

Comment:
yeah make sense, I had another version of this file with the animation faster. Thenks